src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="mac photo"
+        class="card__photo"
+      />
+      <h2 class="card__tittle">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__rate">
+        <div class="card__rate-stars">
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+          <div class="card__rate-stars-star"></div>
+        </div>
+        <div class="card__rate-text">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+      <a
```

Comment:
The 'href' attribute for the 'Buy' button is '#'. It's better to use 'javascript:void(0);' to prevent the page from jumping to the top when the button is clicked.